src/app.py

```diff
@@ -150,6 +150,11 @@ def get_projects(self) -> Dict:
 if "TOPOLOGY_CONFIG" in os.environ:
     app.config.from_envvar("TOPOLOGY_CONFIG", silent=False)
 _verify_config(app.config)
+if "AUTH" in app.config:
+    if app.debug:
+        default_authorized = app.config["AUTH"]
+    else:
+        print("ignoring AUTH option in non-debugging mode", file=sys.stderr)
```

Comment:
Should we use `app.logger` here instead of `print`?

Reply:
Logging hasn't been initialized at that point in the code, so that wouldn't work.

Reply:
So does this get logged in the journal (assuming it's run as a systemd service)?

Reply:
Yeah, it ends up in the httpd logs.